scripts/run_pylint_check.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# This scripts runs the pylint checker on a given input directory
+
+export LC_ALL=en_US.UTF-8
+export LANG=en_US.UTF-8
+
+usage() { echo "Usage: $0 -i <pylint input directory> -o <pylint output directory> [-e <comma seperated excluded directories>]" 1>&2; exit 1; }
+
+while getopts ":i:o:e:" p; do
+    case "${p}" in
+        o)
+            o=${OPTARG}
+            ;;
+        i)
+            i=${OPTARG}
+            ;;
+        e)
+            e=${OPTARG}
+            ;;
+        *)
+            usage
+            ;;
+    esac
+done
+shift $((OPTIND-1))
+
+if [ -z "${i}" ] || [ -z "${o}" ]; then
+usage
+fi
+
+if [ -z "${e}" ]; then IGNORE=""; else IGNORE="--ignore=${e}"; fi
+
+echo "Running pylint with:"
+echo "Input directory: ${i}"
+echo "Output directory: ${o}"
+echo "Ignoring directories: ${e}"
+pylint ${IGNORE} --output-format=parseable --reports=y  ${i} > ${o}/pylint.out 2> ${o}/pylint.log
```

Comment:
Yes, that would be the best, in general the idea is to try to use config vars for everything and document them so they are discoverable. It also makes scripts simpler (you don't have to do all this argument parsing stuff that you did).